docs/cluster-management/profiles.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Profiles
+sidebar_position: 6
+hide_title: true
+---
+
+import TierLabel from "../_components/TierLabel";
+
+<h1>
+  {frontMatter.title} <TierLabel tiers="enterprise" />
+</h1>
+
+:::note BEFORE YOU START 
+The following instructions require you to make minor changes to the content of your own hosted Helm repository.
+:::
+
+When provisioning new clusters it is often useful to install selected software packages to them as part of their bootstrap process. Weave GitOps Enterprise enables this by installing standard Helm charts to the newly provisioned clusters. This feature lowers the ongoing operational overhead and allows for the clusters to be immediately usable after being provisioned. To set this up you need to:
+
+1. Annotate a Helm chart to make it available for installation
+2. Select which profiles you want installed when creating a cluster
+
+### 1. Annotate a Helm chart to make it available for installation
+
+In order for a chart to become available for installation, it needs to include a `weave.works/profile` annotation. For example:
+
+```yaml title="Chart.yaml"
```

Comment:
Suggestion:  Change 'podinfo' for 'observability' , because podinfo is an odd profile, not really a cluster component.